src/server/server.d

```diff
@@ -318,6 +318,19 @@ int main(string[] args)
 			ubyte[] responseBytes = msgpack.pack(response);
 			s.send(responseBytes);
 		}
+        else if (request.kind & RequestKind.localUsage)
+        {
+            try
+            {
+                AutocompleteResponse response = findLocalUsage(request, cache);
+                ubyte[] responseBytes = msgpack.pack(response);
+                s.send(responseBytes);
+            }
+            catch (Exception e)
+            {
+                warning("Could not find local usage", e.msg);
```

Comment:
This is in case something unexpected happens. I don't remember if i written this or if this a converted from the PR that was made where DSymbol was still ACSymbol in DCD.